generator/src/main/java/com/scottlogic/deg/generator/CommandLine/GenerateCommandLine.java

```diff
@@ -10,6 +10,11 @@
 import java.util.List;
 import java.util.Optional;
 
+@picocli.CommandLine.Command(
```

Comment:
I'm sure there is a good reason for this being (re)included, but what is it?
It would be useful to know this to confirm whether the change is 'correct' or not

Reply:
Ignore this comment, just re-read the PR summary.